server/routes/techlist.js

```diff
@@ -0,0 +1,16 @@
+var express = require("express");
+var router = express.Router();
+const db = require("../database/models");
+const tech_list = db.tech_list;
+
+/**
+ * GET for fetching the vote list
+ */
+router.get("/", function (req, res, next) {
+  tech_list
+    .findAll()
+    .then((user) => res.send(user))
```

Comment:
We're not finding users here right? :) I think this should be something like `technologies`?

Reply:
We made note of all your feedback. All these changes will reflect in our next merge. Thank you very much, these tips are very helpful to help us learn code better.